compose.yaml

```diff
@@ -1,24 +1,25 @@
 services:
   hollo:
-    image: ghcr.io/dahlia/hollo:canary
+    image: hollo:canary
```

Comment:
Why was this prefix removed?

Reply:
Ah, that's a leftover from testing a modified image locally. I'll revert it. 😄